SU2_CFD/src/solvers/CMeshSolver.cpp

```diff
@@ -564,6 +567,39 @@ void CMeshSolver::UpdateDualGrid(CGeometry *geometry, CConfig *config){
 
 }
 
+void CMeshSolver::ComputeGridVelocity_FromBoundary(CGeometry **geometry, CNumerics **numerics, CConfig *config){
+
+  /*--- Compute the stiffness matrix, no point recording because we clear the residual. ---*/
+
+  const bool wasActive = AD::BeginPassive();
+
+  Compute_StiffMatrix(geometry[MESH_0], numerics, config);
+
+  AD::EndPassive(wasActive);
+
+  /*--- Clear residual (loses AD info), we do not want an incremental solution. ---*/
+  SU2_OMP_PARALLEL {
+    LinSysRes.SetValZero();
+    LinSysSol.SetValZero();
```

Comment:
In DeformMesh we should a LinSysSol.SetValZero(); for cases with grid velocity otherwise the deformation will use the velocity solution as initial conditions.

Reply:
This should be the reason for one testcase now diverging.